include/pacmod_game_control/publish_control.h

```diff
@@ -48,9 +57,20 @@ class PublishControl
     static int headlight_state;
     static bool headlight_state_change;
     static int wiper_state;
+    static int last_shift_cmd;
```

Comment:
why these variables need to be static?

Reply:
Because of the way the `startup_checks.cpp` is designed. I was thinking to change this, but probably better for another PR since it's not related to this.

Reply:
Ok. Please create another issue to fix this.

Reply:
https://github.com/astuff/pacmod_game_control/issues/85